app/views/evaluators/index.html.erb

```diff
@@ -83,29 +87,37 @@
         <table aria-label="List of Evaluators" class="usa-table usa-table--stacked-header usa-table--borderless width-full gray-header">
           <thead>
             <tr>
-              <th scope="col">Evaluator name</th>
+              <th scope="col">Name</th>
               <th scope="col">Email address</th>
               <th scope="col">User role</th>
               <th scope="col">User status</th>
-              <th scope="col">Assigned submissions</th>
-              <th scope="col"><span class="usa-sr-only">Actions</span></th>
+              <th scope="col"># of assigned submissions</th>
```

Comment:
columns should be capitalized title-cased:
- Email Address
- User Role
- User Status
- # of Assigned Submissions